### ISSUE TITLE
Console error when sending feedback while there's a problem with a TN check

### ISSUE THREAD

**Issue by birchamp** [unfoldingWord/translationCore#7013]
### Description

There's a console error for the feedback feature when there is a problem with a tN check. It errors when the user chooses ignore, or goes into send feedback and clicks cancel.

Example screenshot:
![image.png](https://images.zenhubusercontent.com/5cf53901e39e5f33e3e129fa/2da61cb0-fd31-459b-9e0d-001e9fff31fb)
### Steps to Reproduce

**Screenshots**

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

### Context

**Enviorment**
| Tech         | Version |
|--------------|---------|
| tc           | v0.?.?  |
| build #      |         |
| OS           |         |

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by cckozie** [comment]
I'm not seeing an actual failure in 3.0.0 (32e54bf) 
I see this error in the console after clicking Ignore on the Problem with check alert, or clicking Cancel on the Feedback form. No indication to the user that there is a problem, and sending Feedback works correctly.
![image](https://user-images.githubusercontent.com/8171759/85246210-a55bfe00-b40f-11ea-915c-67b424bcf47b.png)

**Comment by PhotoNomad0** [comment]
@cckozie - it is basically the same error, just you are seeing in the compiled code.  I think the main problem to the user is that it does not automatically move to the next check.

**Comment by PhotoNomad0** [comment]
@cckozie - nevermind - it looks like it is auto advancing

**Comment by PhotoNomad0** [comment]
**Note:**
It looks like the cause is that in VerseCheckWrapper we are passing a dispatch:
```
     onInvalidCheck: () => {
      onInvalidCheck(contextId, gatewayLanguageCode, true, dispatch(changeToNextContextId(projectSaveLocation, userData, gatewayLanguageCode, gatewayLanguageQuote)));
    },
```

**Comment by PhotoNomad0** [comment]
Verified in translationCore 3.0.0 (07e320a)

**Comment by cckozie** [comment]
Fixed 3.0.0 (a03e2cc)
